tests/acceptance/features/webUISharingInternalUsersBlacklisted/shareWithUsers.feature

```diff
@@ -1,4 +1,5 @@
-@issue-ocis-1317
+@issue-ocis-1317 @skipOnOCIS
+# exclude-groups-from-sharing configuration is not available now in the oCIS webUI
```

Comment:
wasn't available previously either (AFAIK)
```suggestion
# exclude-groups-from-sharing configuration is not available in the oCIS
```

Reply:
@saw-jan I think `now` is two edge sword. This configuration will be added later on ocis but for now it is not available

Reply:
Oh, I read it differently. But I think suggestions are good though :smile: